crates/sui-core/src/unit_tests/narwhal_manager_tests.rs

```diff
@@ -216,3 +219,51 @@ async fn test_narwhal_manager() {
         _ = tr_shutdown.send(());
     }
 }
+
+#[tokio::test]
+async fn test_remove_old_epoch_data() {
```

Comment:
Probably this test now fits better under the `epoch_data` test and not under the `narwhal_manager_tests` ?